grabbit/src/main/groovy/com/twcable/grabbit/resources/GrabbitResourceProvider.groovy

```diff
@@ -72,14 +61,16 @@ class GrabbitResourceProvider implements ResourceProvider {
     @Override
     Resource getResource(ResourceResolver resolver, String path) {
         switch (path) {
-            case ~/^${GRABBIT_JOB}$/:
-                log.debug "Called for path : ${path}"
-                return getJobResource(resolver, path, GRABBIT_JOB_RESOURCE_TYPE)
-            case ~/^${GRABBIT_JOB}\/(.+).(html|json)$/:
-                log.debug "Called for path : ${path}"
-                return getJobResource(resolver, path, GRABBIT_JOB_RESOURCE_TYPE)
+            case ~/^\/grabbit\/job(\/.*)?$/:
+                log.debug "Resolving ${path} to JobResource"
+                return new JobResource(resolver, path)
+            case ~/^\/grabbit\/transaction(\/.*)?$/:
+                log.debug "Resolving ${path} to TransactionResource"
+                return new TransactionResource(resolver, path)
+            case ~/^\/grabbit\/content(\/)?$/:
+                log.debug "Resolving ${path} to ContentResource"
+                return new ContentResource(resolver, path)
             default:
-                log.warn "Unable to find resource for path: ${path}."
```

Comment:
Indeed. It's definitely not a great response, but it wouldn't say it's incorrect - a resource really can't be found for /grabbit. Ideally, we would create a parent resource type for /grabbit, a root resource servlet, and respond in the spirit of HATEOAS by forwarding the request to a JSP generated usage page.  We have a ticket for this in https://github.com/TWCable/grabbit/issues/22